### DISCUSSION TITLE
GitHub Copilot support

### DISCUSSION THREAD

**Discussion by bigint** [zed-industries/zed#6525] [Feature Requests]
It would be great if zed support copilot, nowadays they play a crucial role in development IMO.

**Comment by JosephTLyons** [comment]
Hey @bigint, we certainly want to support copilot.  The team is having discussions about how to do so currently.  Until then, here's an issue that you can upvote (👍) to show you want this feature:

- https://github.com/zed-industries/community/issues/301

**Reply by gobijan** [reply]
Copilot is very important nowadays. The vim plugin is exposing a node based LSP that can be used for editor neutral integration. Sublime has copilot through this and it works as good as vscode.

Here is the link to the sublime extension:
https://github.com/TerminalFi/LSP-copilot

**Reply by samgaw** [reply]
Is there an idiot's guide on test driving [zed-industries/copilot](https://github.com/zed-industries/copilot) or is it waiting for a new binary release? Thanks.

**Reply by JosephTLyons** [reply]
> Is there an idiot's guide on test driving [zed-industries/copilot](https://github.com/zed-industries/copilot) or is it waiting for a new binary release? Thanks.

Waiting on a binary release - it should be soon 😃

**Comment by JosephTLyons** [comment]
Zed now has both Copilot and Superman (WIP) support.
